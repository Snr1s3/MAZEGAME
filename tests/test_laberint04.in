gAAAAABmL4AjAR84CH15CzNM6Ci6urRRpL05wk3g0No1fpj8_0G0amWxPWNoj105kNoeVWnSY5EtLGpCAkz1OWBh8WmPIcMbf0_2bHqWm5LfIjycb0AL1mOqA_9y-TbPvoc4nZXAg9ZA