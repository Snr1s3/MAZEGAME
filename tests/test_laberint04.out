gAAAAABmL4AjWSPbtwpoctbFJ9oWmS_Bca6VLOzh-zOWdHWbatGT_xV3DWdaC9J5EZj6ROQhy-sR1_cFAVWcYpGlefMxPdNbEwgGZRySAFLDoUNnM3RmElvCFKGp6HaXmf1ayvVnFcOPU0aNEInIH_vTWWYR3ycy4I7Cy4zrtkuHDmw9jkPgfnP8gJYQm-SMlIZN9g1zsNe1z6vqCjloN5VDzyHWQqsEEwSi_qJwXc17PDAuAXODUIsveNJhco1XfK_ZUeIQrb6PiSqpmzbAo6LxEuZSAktfvhDSCPVyXrWJDD4aBXZts0bv9ci-20e7S2UZr9rp6wqyu_G-AuS6aZHidarOgHC5qMXDIiWPKqidSyizZNo7zbhJwH16RhwzhhKIqdOnm800gdDtq7Xr_oGDq1CtzB3aWr8suEcozP3HG8zHaA8iq0aupYfvNMzzSsTH3i3hqNkn4cmiylRMHMa_plMf6iUGtCUvpFda5l6biWm68h6ZEeAsFUt-3GDL2OGarNwhLjU4l39eCtgOd6ogAu4aAwu8DgsLwzy8aSOJNr_cs90e2tOZZJCUJ1Sr9e3ijGu6OR6jfw-sAHhJ3xeuBYrkuSFnxYN__p1lC31NK_IH3UU-XeCDcNr3YEFFi_Hqdc-qEFhWtHgeaUchlj2EG_p2TDq_YlCdcCE0kFR3yDAcBNk1-yv628bhOTkvczq9vMWFMlh4gHy9BjHiBfOL9YdHhlthBzyVGdsMlqAEMsem6V48W9j9Ond-f9beWAU87iAa_EpXaleXCZDvSlwHG-ibKq5Dgu7-LSrJeCY2748MJiDsiIq_vI0hLJSNKE8Ah5pwqnNkcHBVwaFZS4Ay22uXQNGiWB6iL1YuC7uFuy5k56XB5EnhTWZyTy7xXstfpbaN1F2pbgcn1L8hDu-HlaPM0xqdSovF_pLJsDzMtv8xkWRZceNrGie_gsbeehvd0ak5rl-RGEmBXcCRe43FtCbK3qzaZ47G_VbSD4jKilIB1O86N0V_oW-_Kx-4SqHzCOW-PwTfpmMMUgT6hSddooPvNtOW-DTsHy034KByH-kSSJwk9bWNeMdKs_UoxSPwuyBtyIus4BueHxv1nrNAQ0c5L6ZA7nCEUOdnToPc2kLnJU3n12c=